### ISSUE TITLE
WCAG issue - Video and audio files cannot be paused when reviewing them from the Pending tab

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#16]
Submitted in Kaltura Support Case #00548907:

When accessing a course's Media Gallery where files are pending instructor review for approval or rejection, if a user selects an audio or video file's thumbnail to preview it, the media cannot be paused. The media can be muted and forwards and backwards seeking can be used, but there is no way to pause the player, creating a failure of WCAG Success Criterion 2.2.2: Pause, Stop, Hide (Level A). All users should be able to pause media when previewing it in the Pending tab of a Media Gallery.

Steps to reproduce:
1. Access a course's Media Gallery where there are video or audio file(s) pending instructor review and approval.
2. Select the Pending tab.
3. Press the Play button on the video/audio file's thumbnail. Notice that you can mute the media file or skip forwards or backwards, but the media file cannot be paused.

**Comment by judyucb** [comment]
UCB case opened on 5/21: #00548741 "Player available on Pending tab of MG has no way to pause"

**Comment by judyucb** [comment]
Update (as of ten minutes ago): "It looks like Tier 3 has reviewed the issue and proposed a fix. I’ve requested a new update from them and will get back to you as soon as I hear back."
